providers/zoom/session_test.go

```diff
@@ -0,0 +1,47 @@
+package zoom_test
+
+import (
+	"github.com/markbates/goth"
+	"github.com/markbates/goth/providers/zoom"
+	"github.com/stretchr/testify/assert"
+	"testing"
```

Comment:
Same here, the imports should be grouped as follows.

```suggestion
	"testing"
	
	"github.com/markbates/goth"
	"github.com/markbates/goth/providers/zoom"
	"github.com/stretchr/testify/assert"
```